cmd/result.go

```diff
@@ -44,25 +42,24 @@ func (res Result) Print() {
 	}
 }
 
-func createFields(res Result, err int) (fields log.Fields) {
+func createFields(res Result, occ Occurrence) (fields log.Fields) {
 	fields = log.Fields{}
 	v := reflect.ValueOf(res)
-	for _, member := range shouldLog(err) {
+	for _, member := range shouldLog(occ.id) {
 		value := v.FieldByName(member)
 		if value.IsValid() && value.Interface() != nil && value.Interface() != "" {
 			fields[member] = value.Interface()
 		}
 	}
+	for k, v := range occ.metadata {
```

Comment:
So `metadata` will always be logged, right? Should we add a comment/change the name then? Just so that it's purpose is obvious to other people that come and work on this repo.

Reply:
Metadata will only be logged if it is specified in the occurrence (otherwise `occ.metadata` will be nil and the for loop wont execute). 
Open to changing the name, I just picked metadata because its a store for all the miscellaneous fields that not all occurrences need to have. Maybe `fields`?

Reply:
`fields` is fine, take a look at @jinankjain suggestion for making it its own `type` with json serialization.